packages/manager/modules/billing/src/payment/method/add/views/legacyBillingAddress/controller.js

```diff
@@ -28,10 +34,26 @@ class BankAccountModel {
 }
 
 export default class PaymentMethodAddLegacyBillingAddressCtrl {
+  /* @ngInject */
+  constructor(coreConfig) {
+    this.coreConfig = coreConfig;
+  }
+
   $onInit() {
     this.model.billingAddress = new BankAccountModel(
       this.currentUser.billingCountry,
     );
+    this.setDataProtectionLinks();
+  }
+
+  setDataProtectionLinks() {
+    const currentLanguage = this.coreConfig.getUserLanguage();
```

Comment:
should this be based on subsidiary? if so you need to use FR and GB instead of fr and en

Reply:
I like this approach more than using subsidiary. This gives a flexibility of choosing the language and all docs irrespective of subsidiary.